protobuf-rust/plan.sh

```diff
@@ -1,16 +1,19 @@
 pkg_name=protobuf-rust
 pkg_origin=core
-pkg_version=1.0.18
+pkg_version=1.4.4
 pkg_license=('BSD')
 pkg_source=nosuchfile.tar.gz
 pkg_bin_dirs=(bin)
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_upstream_url="https://github.com/stepancheg/rust-protobuf"
+pkg_description="Rust implementation of Google protocol buffers"
 pkg_deps=(core/glibc core/gcc)
-pkg_build_deps=(core/rust core/gcc core/cacerts core/protobuf)
+pkg_build_deps=(core/rust core/gcc core/cacerts core/protobuf-cpp)
 
 do_build() {
-  env SSL_CERT_FILE=$(pkg_path_for cacerts)/ssl/cert.pem \
-      PROTOBUF_PREFIX=$(pkg_path_for protobuf) \
-      cargo install protobuf --root $pkg_prefix --vers $pkg_version -j$(nproc) --verbose
+  env SSL_CERT_FILE="$(pkg_path_for cacerts)/ssl/cert.pem" \
+      PROTOBUF_PREFIX="$(pkg_path_for protobuf)" \
```

Comment:
This prints a WARN in the build since `core/protobuf` is not a dependency anymore. Also, I'm wondering it it is useful at all. No mention of `PROTOBUF_PREFIX` seem to be found in both protobuf and protobuf-rust sources.